build_linux.sh

```
@@ -3,6 +3,8 @@ set -euxo pipefail
 
 echo "Building cryptomator cli..."
 
+APP_VERSION='0.1.0-local'
```

Shellcheck (0.9.0-1)

[warning] APP_VERSION appears unused. Verify use (or export if used externally).

(SC2034)